presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,82 @@
+* Add warnings for :func:`approx_distinct` and :func:`approx_set` with low ``MAX_STANDARD_ERROR``.
+* Add support for retrying queries that failed with max requests queued exception.
+* Update protoc and grpc version in presto-grpc-api module to adapt to linux aarch64.
+* Upgrade zstandard compression to version 1.5.2.2. This improves the cpu used for data compressed with zstandard by about 2%.
+* Upgrade postgresql driver to 42.3.3.
+
+Clickhouse Changes
+____________
+* Add a Presto connector for Clickhouse with support of username/password authentication.
+
+Hive Changes
+____________
+* Add a limit that prevents caching partitions with column counts greater than a configured threshold. This threshold can be set using the ``hive.partition-cache-column-count-limit`` configuration parameter.
+* Add metastore config property ``hive.metastore.thrift.delete-files-on-table-drop`` to delete files on drop table.
+* Add support for overwriting existing partitions with a hive config property ``hive.insert-existing-partitions-behavior``. The config supersedes a legacy config ``hive.insert-overwrite-immutable-partitions-enabled``. The new config adds capability of overwriting new partitions for S3.
+* Add support for timely Parquet metadata cache invalidation.
+* Replace Hive session property ``streaming_aggregation_enabled`` with ``order_based_execution_enabled``.
+* Replace Hive config ``hive.streaming-aggregation-enabled`` with ``hive.order-based-execution-enabled``.
+
+Iceberg Changes
+_______________
+* Fix Iceberg ``$files`` table in case of column dropping.
+* Add ``$properties`` system table.
+* Add support for storing column comments for Iceberg tables.
+* Upgrade Iceberg to 0.13.1.
+
+Mongodb Changes
+_______________
+* Add :func:`CAST(ObjectId() as STRING)`.
+
+Pinot Changes
+_____________
+* Improve query performance by enabling pushdown of topn broker queries by default.
+* Add double-quotes to Pinot generated queries to ensure that reserved keywords are escaped.
+* Add TLS support in Pinot gRPC connection.
+* Upgrade Pinot connector to support Pinot 0.10.0.
+
+Spark Changes
+_____________
+* Improve handling of scenario where a query fails due to broadcast table violating node memory limit or broadcast limit. Previously the query failed while workers tried to load broadcast table causing container out of memory.
+* Add a new configuration property ``spark.retry-on-out-of-memory-broadcast-join-enabled`` to disable broadcast join on broadcast OOM and retry the query again within the same spark session.  This can be overridden by ``spark_retry_on_out_of_memory_broadcast_join_enabled`` session property.
+
+Verifier Changes
+________________
+* Add support to verifier for running control and test queries concurrently through config ``concurrent-control-and-test`` and test property.
```

Comment:
Add support to verifier for running control and test queries concurrently by setting ``concurrent-control-and-test`` configuration property to true.